modules/apps/foundation/frontend-js/frontend-js-aui-web/src/main/resources/META-INF/resources/liferay/session.js

```diff
@@ -505,7 +505,7 @@ AUI.add(
 						instance._intervalId = host.registerInterval(
 							function(elapsed, interval, hasWarned, hasExpired, warningMoment, expirationMoment) {
 								if (!hasWarned) {
-									instance._uiSetActivated();
+									instance._host.extend();
```

Comment:
instance._host is defined at line 479

Reply:
I saw. So I'll change to `host.extend();`

Reply:
I fixed it.